packages/perseus/src/widgets/orderer/orderer.util.ts

```diff
@@ -0,0 +1,30 @@
+import type {
+    PerseusOrdererWidgetOptions,
+    PerseusRenderer,
+} from "@khanacademy/perseus-core";
+
+/**
+ * For details on the individual options, see the
+ * PerseusOrdererWidgetOptions type
+ */
+type OrdererPublicWidgetOptions = {
+    options: ReadonlyArray<PerseusRenderer>;
+    height: "normal" | "auto";
+    layout: "horizontal" | "vertical";
```

Comment:
I'm not sure if this was discussed elsewhere, but it might be useful to build these types off of the main widget options type. This will help keep it clear that this type is a subset of the full widget options. 

```suggestion
    height: PerseusOrdererWidgetOptions["height"];
    layout: PerseusOrdererWidgetOptions["layout"];
```

Reply:
Updated :)